src/test/java/Test.java

```diff
@@ -0,0 +1,32 @@
+import java.util.List;
+import org.junit.Assert;
+
+public class Test {
+    private final Double delta = 0.0;
+
+    @org.junit.Test
```

Comment:
```suggestion
    @Test
```
everywhere

Reply:
It doesn't allow me to make it test, the compiler doesn't let it go